core/src/main/java/org/mapfish/print/cli/CliDefinition.java

```diff
@@ -45,6 +45,9 @@
             "overriding certain values if desired")
     public String springConfig = null;
 
+    @Argument(description = "Use standard LogBack log configuration and don't override with internal config files.")
+    public boolean uselogback = true;
```

Comment:
Isn't better ton name it with something like `configureLogback`...

Reply:
Remark: This is a WIP pull request, it is more for the Proxy than the LogBack. When finished with the proxy, I will open a clean PR for the LogBack. Thanks for reviewing anyways.